src/rt/dmain2.d

```diff
@@ -558,6 +558,35 @@ private void formatThrowable(Throwable t, scope void delegate(in char[] s) nothr
     }
 }
 
+private auto parseExceptionOptions()
+{
+    static struct ExceptionOptions
+    {
+        bool trap;
+
+        @nogc nothrow void initialize()
+        {
+            import rt.config : rt_configOption;
+            trap = rt_trapExceptions;
+            auto res = rt_configOption("extrap");
+            switch(res)
+            {
+                case "1", "y", "Y", "on", "true":
```

Comment:
`core.internal.parseoptions` only handles `"1", "y", "Y"`, but I think "on" and "true" are too common to ignore.

Reply:
Maybe just add `on`and `true` there, too? Please also note that parse only checks the first character, so it allows `yes`, too.

Reply:
> but I think "on" and "true" are too common to ignore.

I think this is too clever for the value it adds. Note that "On" and "True" will be ignored as well. If you want to prevent mistakes, throw when the option is not recognized. e.g.: https://github.com/dlang/druntime/blob/522787e95062398ad96dbf35e3983c10dd370c9a/src/rt/minfo.d#L178